### ISSUE TITLE
Surface parameter has been removed from MuonGunWeightCalculator but it's in the docs (Trac #1627)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1619]
The WeightCalculatorModule does not accept "Surface" as parameter, so this doc is wrong: 
http://software.icecube.wisc.edu/simulation_trunk/projects/MuonGun/weighting.html

Migrated from https://code.icecube.wisc.edu/ticket/1627
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:12:58", 
    "description": "The WeightCalculatorModule does not accept \"Surface\" as parameter, so this doc is wrong: \nhttp://software.icecube.wisc.edu/simulation_trunk/projects/MuonGun/weighting.html", 
    "reporter": "delia.tosi", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067178841456", 
    "component": "other", 
    "summary": "Surface parameter has been removed from MuonGunWeightCalculator but it's in the docs", 
    "priority": "minor", 
    "keywords": "", 
    "time": "2016-04-12T14:27:26", 
    "milestone": "", 
    "owner": "jvansanten", 
    "type": "cleanup"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160412T14:30:56: nega commented:

> Good catch. Please update the docs in svn or submit a patch.

**Comment by icecube-trac** [comment]
Trac update at 20160425T18:01:46: olivas commented:

> Please set a milestone.

**Comment by icecube-trac** [comment]
Trac update at 20160503T13:22:14: olivas changed milestone from "" to "May 2016"

**Comment by icecube-trac** [comment]
Trac update at 20160729T06:38:28: 
* jvansanten changed status from "new" to "closed"
* jvansanten changed resolution from "" to "fixed"
* jvansanten commented:

> In changeset 148883:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="148883"
> Fix #1619
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:12:58: 
* olivas changed milestone from "May 2016" to ""
* olivas commented:

> Milestone May 2016 deleted
